homeassistant/generated/integrations.json

```
@@ -6871,6 +6871,12 @@
       "config_flow": false,
       "iot_class": "cloud_polling"
     },
+    "watergate": {
+      "name": "Watergate",
+      "integration_type": "hub",
```

```suggestion
      "integration_type": "device",
```
looks like this integration only provides a single devices at a time